s3tokenizer/model_v2.py

```
@@ -19,7 +19,7 @@
 from einops import rearrange
 
 from s3tokenizer.model import Conv1d, LayerNorm, Linear, MultiHeadAttention
-from s3tokenizer.utils import make_non_pad_mask, mask_to_bias, onnx2torch
+from s3tokenizer.utils import make_non_pad_mask, mask_to_bias, onnx2torch, merge_tokenized_segments, padding
```

Ruff (0.11.9)

`s3tokenizer.utils.padding` imported but unused

Remove unused import: `s3tokenizer.utils.padding`

(F401)